esmgrids/cice_grid.py

```diff
@@ -135,7 +139,11 @@ def write(self, grid_filename, mask_filename, metadata=None):
         angle.standard_name = "angle_of_rotation_from_east_to_x"
         angle.coordinates = "ulat ulon"
         angle.grid_mapping = "crs"
-        angleT = self._create_2d_nc_var(f, "angleT")
+
+        if variant == "cice5-auscom":
+            angleT = self._create_2d_nc_var(f, "angleT")
+        else:  # variant==cice6
```

Comment:
I'm assume you chose to do the switching with a string, rather than a binary `cice6 = False` because you want to allow for the potential for other variants in the future? If so, it might be worth explicitly doing `elif variant == "cice6"` and producing an error or warning otherwise. Or if it's only ever going to be binary (ie cice5-auscom or cice6) then the API should reflect that.

Reply:
Ok. Ill make in a boolean ... its hard to be sure tho. There's 80% chance cice7 would use 'anglet' but maybe it won't, or we find some other fork we need to support that uses a different name.

Reply:
Then maybe safest to leave as is and issue an error or warning (e.g. "unrecognised variant, defaulting to cice5-auscom") if it's not a recognised string?

Reply:
I want the default to be cice6, but I dont want to change the current interface. So if a variant is provided its set to cice6, if no variant then its cice5-auscom. A bit confusuing maybe. Let me know :)

Reply:
Yeah I think that's a bit confusing. I think the default for an unrecognised variant should be the same as the method default. You could just make the method default `variant="cice6"` and for backwards compatibility someone would have to set `variant="cice5-auscom"`? That'll be more convenient for us going forward anyway right?

Reply:
Yeah that makes sense if you are happy with that? It just breaks back-ward compatibility?

Reply:
I think I'm fine with that. As far as we know all existing uses of this package are via a git submodule so won't be a problem.

Reply:
Ok cice6 is now by default, cice5-auscom is optional. It raises an error on any other variant that is not cice5-auscom.